frontend/src/app/modules/interventions/details/comments/comments.component.html

```diff
@@ -0,0 +1,20 @@
+<section *ngFor="let comment of comments" class="comment">
+  <div class="comment-text">{{ comment.comment }}</div>
+  <div class="comment-footer">
```

Comment:
Do we need both `footer` and `div.comment-footer` wrapper?

Reply:
Yeah, indeed, not needed. Removed 👍